### PR TITLE
Add/amp anim for gif

### LINKED ISSUE #72: gifs should be wrapped in <amp-anim>
https://github.com/ampproject/amphtml/tree/master/extensions/amp-anim

### PR DESCRIPTION
**Pull request by j3j5**
Parse `.gif` images into `<amp-ani>` tags and add the proper script script to the scripts content array.
It should fix #72

### CODE REVIEW

File: class-amp-content.php [hunk 1]
```diff
@@ -32,12 +32,15 @@ public function transform() {
...
 
+		$img_converter = new AMP_Img_Converter( $content );
 		// Convert HTML to AMP
 		// see https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md#html-tags)
-		$content = $this->convert( new AMP_Img_Converter( $content ), array(
+		$content = $this->convert( $img_converter, array(
 			'layout' => 'responsive',
 		) );
 
+		$this->add_scripts( $img_converter->get_scripts() );
```
Comment (mjangda) [inline class-amp-content.php]: The `convert` method calls `add_scripts` for us so we shouldn't need to call it here.

File: class-amp-img.php [hunk 2]
```diff
@@ -5,6 +5,11 @@
 class AMP_Img_Converter extends AMP_Converter {
 	public static $tag = 'img';
 
+	private $anim_extensions = '/.(gif|gifv)/';
```
Comment (mjangda) [inline class-amp-img.php]: Does `gifv` really makes sense here? (in most cases, it serves up an HTML file)

**Comment by mjangda** [comment]
Thanks for this PR @j3j5! Left two comments inline.

File: class-amp-img.php (hunk 2 above)
Comment (j3j5) [inline class-amp-img.php]: Yeah, I wasn't really sure, I actually asked in [SO](https://stackoverflow.com/questions/34501955/what-are-the-supported-formats-for-amp-anim) about what file formats should `<amp-anim>` be used with, but to no avail so far. I can remove it for `.gif` only and we can add more formats if necessary later.

File: class-amp-content.php (hunk 1 above)
Comment (j3j5) [inline class-amp-content.php]: True, Idk how I looked at it I thought they weren't being added.

**Comment by j3j5** [comment]
I've fixed the changes from your comments.

**Comment by mjangda** [comment]
Made a few small tweaks and merged in manually. Thanks again @j3j5!

**Pull request merged**